x-pack/plugins/secops/public/pages/home/index.tsx

```diff
@@ -79,82 +61,83 @@ const calculateFlyoutHeight = ({
+  <AutoSizer detectAnyWindowResize={true} content>
+    {({ measureRef, windowMeasurement: { height: windowHeight = 0 } }) => (
+      <WrappedByAutoSizer data-test-subj="wrapped-by-auto-sizer" innerRef={measureRef}>
+        <Page data-test-subj="pageContainer">
+          <DragDropContextWrapper>
+            <Flyout
+              flyoutHeight={calculateFlyoutHeight({
+                additionalFlyoutPadding: additionalEuiFlyoutPadding,
+                globalHeaderSize: globalHeaderHeightPx,
+                windowHeight,
+              })}
+              headerHeight={flyoutHeaderHeight}
+              timelineId="timeline-1"
+            >
+              <StatefulTimeline
+                flyoutHeaderHeight={flyoutHeaderHeight}
                 flyoutHeight={calculateFlyoutHeight({
                   additionalFlyoutPadding: additionalEuiFlyoutPadding,
                   globalHeaderSize: globalHeaderHeightPx,
                   windowHeight,
                 })}
-                headerHeight={flyoutHeaderHeight}
-                timelineId="timeline-1"
-              >
-                <StatefulTimeline
-                  flyoutHeaderHeight={flyoutHeaderHeight}
-                  flyoutHeight={calculateFlyoutHeight({
-                    additionalFlyoutPadding: additionalEuiFlyoutPadding,
-                    globalHeaderSize: globalHeaderHeightPx,
-                    windowHeight,
-                  })}
-                  id="timeline-1"
-                  headers={headers}
-                />
-              </Flyout>
+                id="timeline-1"
+                headers={headers}
+              />
+            </Flyout>
+            <EuiPageBody>
               <PageHeader data-test-subj="pageHeader">
-                <HeaderFlexGroup justifyContent="spaceBetween" alignItems="center">
-                  <EuiFlexItem grow={false} data-test-subj="datePickerContainer">
-                    <Navigation data-test-subj="navigation" />
-                  </EuiFlexItem>
-                  <EuiFlexItem grow={false}>
-                    <EuiFlexGroup alignItems="center" wrap={false}>
-                      <EuiFlexItem grow={false} data-test-subj="datePickerContainer">
-                        <RangeDatePicker id="global" />
-                      </EuiFlexItem>
-                      <EuiFlexItem grow={false} data-test-subj="appSettingsContainer">
-                        <AppSettings />
-                      </EuiFlexItem>
-                    </EuiFlexGroup>
-                  </EuiFlexItem>
-                </HeaderFlexGroup>
+                <PageHeaderSection>
+                  <EuiFlexGroup justifyContent="spaceBetween" alignItems="center" gutterSize="m">
+                    <EuiFlexItem grow={false} data-test-subj="datePickerContainer">
+                      <Navigation data-test-subj="navigation" />
+                    </EuiFlexItem>
+                    <EuiFlexItem grow={false} style={{ marginTop: '13px' }}>
```

Comment:
Really we should strive for not using styled components and avoid inline styles, but I'm fine marking these from now on as optional. I doubt I'll run through the code base replacing them anymore.

Reply:
I think some of the issues and reasons are that the EuiComponents don't interact well with them and they don't use them much? They use a different CSS style?

Reply:
oh, I will fix that, it was laziness from me. thanks to catch it